doc/design/model_zoo_pai.md

```diff
@@ -0,0 +1,134 @@
+# Model Zoo on Alibaba PAI
+
+## Introduction
+
+[Alibaba PAI](https://www.alibabacloud.com/product/machine-learning) is an end-to-end platform that provides various machine learning algorithms to meet users' data mining and analysis requirements. PAI supports TensorFlow and many other machine learning frameworks. SQLFlow model zoo also works on PAI. [Model Zoo Design Doc](model_zoo.md) is a high level design for SQLFlow model zoo. This document is about how to do model training, model prediction, and model analysis using SQLFlow model zoo on PAI.
+
+## Background
+
+### Submit a Task to PAI
+
+A PAI program is a python program that uses TensorFlow or other frameworks like PyTorch. PAI requires the python program to expose several command line options to communicate with [MaxCompute](https://www.alibabacloud.com/product/maxcompute) and OSS([Alibaba Cloud Object Storage Service](https://www.alibabacloud.com/product/oss)). Typically, a user can submit the PAI program by:
+
+1. Use the MaxCompute CLI program [`odpscmd`](https://www.alibabacloud.com/help/doc-detail/27971.htm):
+
+    ```bash
+    $ odpscmd -e "pai -name tensorflow -Dscript=file:///my/training/script.py -Dother_param=other_param_value..."
+    
+    ```
+
+    When a user runs the above command, the CLI will submit the task to PAI then print a logview URL and an instance ID to stdout, after which the user can safely terminate the CLI. The user can check the logview to get the running status of the task or run:
+    
+    ```bash
+    $ odpscmd -e "wait instanceID"
+    ```
+    
+    to attach to the running task.
+
+1. Use the MaxCompute Python Package [`pyodps`](https://pyodps.readthedocs.io/en/latest/):
+
+    ```python
+    from odps import ODPS
+    
+    conn = ODPS(...)
+    inst = conn.run_xflow('tensorflow', parameters={'script':      'file:///my/training/script.py',
+                   	                                'other_param': 'other_param_value' ... })
+    print(instance.get_logview_address())
+    ```
+    
+    This code snippet asynchronously submits a task to PAI and prints the logview URL to stdout.
+
+**Note** that the code snippets here are only for demonstration. In practice, there may be multiple python scripts in a PAI program. If so, we can pack all the python scripts/dependencies of the PAI program into a tarball that can be passed to the `-Dscript` option. PAI will automatically unpack the tarball. And PAI has another command line option to specify which python script is the entry of the program. For details, please refer to the user guide of [odpscmd](https://www.alibabacloud.com/help/doc-detail/27971.htm) and [pyodps](https://pyodps.readthedocs.io/en/latest/).
+
+### Train/Predict/Analyze in SQLFlow with Model Zoo
```

Comment:
I think this section has been explained in the model zoo design doc, can we delete this section?